src/transform/class/PotentialClass.js

```diff
@@ -93,6 +100,38 @@ class PotentialClass {
   }
 
   createConstructor() {
-    return this.constructor.isEmpty() ? undefined : this.constructor.toMethodDefinition();
+    if (this.constructor.isEmpty()) {
+      return undefined;
+    }
+    else {
+      this.modifySuperCalls();
+      return this.constructor.toMethodDefinition();
+    }
+  }
+
+  modifySuperCalls() {
+    this.constructor.methodNode.body.body.forEach(body => {
```

Comment:
Here we break encapsulation of `PotentialMethod` class - we reach deep inside the `methodNode` property and modify it. Instead we should ask `PotentialMethod` to modify itself.